lib/action.js

```diff
@@ -72,7 +73,8 @@ switch(query) {
       .catch(console.log);
   break;
   case 'transition':
-    let [ticketId, action, token] = args;
+    let [ticketId, action] = args;
+    token = keychain.find()
```

Comment:
This is your user's basic auth token and it isn't necessary to pass it here. The token that is expected by the `.transition` method is the [`atl_token`](https://community.atlassian.com/t5/Answers-Developer-Questions/quot-atl-token-quot-in-URL/qaq-p/575268).